source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/TextBlockRenderer.java

```diff
@@ -158,6 +158,11 @@ else if ((textBlock = TextBlock.dynamic_cast(baseCardElement)) == null)
             throw new InternalError("Unable to convert BaseCardElement to TextBlock object model.");
         }
 
+        if (textBlock.GetText().isEmpty())
```

Comment:
is a string of spaces valid?

Reply:
We haven't decided it yet. fwiw, JS doesn't render the space or textblock but renders its separator in such case.

Reply:
Right, we said spaces are the way to force-show an empty TextBlock, pending further discussions..

Reply:
Were you able to test this out with the samples that @jwoo-msft checked in with his change? 

---
In reply to: [447998584](https://github.com/microsoft/AdaptiveCards/pull/4262#discussion_r447998584) [](ancestors = 447998584)